cmd/flags.go

```diff
@@ -54,6 +54,14 @@ var globalFlags = []cli.Flag{
 		Name:  "insecure",
 		Usage: "Disable SSL certificate verification.",
 	},
+	cli.IntFlag{
+		Name:  "older-than",
+		Usage: "Select objects older than N days",
+	},
+	cli.IntFlag{
+		Name:  "newer-than",
```

Comment:
> If these are global flags shouldn't they be not displayed with other commands which don't support this? like mc share mc ls mc events ...
> 
> We should only add these flags for the relevant commands which do support it.

Yes, as I remember that was @abperiasamy proposal. I also don't recommend having it as a global options because it causes more confusion for the user and we don't have any benefits in the other side.